control/trajectory_follower_nodes/design/longitudinal_controller-design.md

```diff
@@ -148,7 +148,7 @@ AutonomouStuff Lexus RX 450h for under 40 km/h driving.
 
 | Name                                 | Type   | Description                                                                                                                                                                             | Default value |
 | :----------------------------------- | :----- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
-| control_rate                         | double | control rate [Hz]                                                                                                                                                                       | 30            |
+| longitudinal_ctrl_period             | double | longitudinal control period [s]                                                                                                                                                         | 0.03          |
```

Comment:
There is a 10% change between a control rate of `30Hz` and a period of `0.03s` (~`33Hz`).
I think it is better to put the period to a value of `0.0333s` to reduce the change to 0.1%.

Reply:
The reason I did not change period to 0.0333 is mentioned in #701 . The original parameter of 33.33333 ms in period is not a possible period for real vehicle control interface. So I chose 30ms instead which has the minimum gap and also consistent with real vehicle interfaces.